app/services/csv_import_service.rb

```diff
@@ -0,0 +1,18 @@
+class CsvImportService
+    require 'csv'
+  
+    def call(file)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/1d081ecd36c48dd47f0123e7164aa066b7a4d685/app/services/csv_import_service.rb#L4"></span>Doesn't depend on instance state (maybe move it to another class?)</td>
    </tr>
  </tbody>
</table>